cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,261 @@ Cask Data Application Platform Release Notes
+  Fixed an issue with the documentation of the Data Quality Application's "Unique Value" function.
+
+- `CDAP-4145<https://issues.cask.co/browse/CDAP-4145>`__ -
+  Added documentation of the new view and stream-views feature.
+
+- `CDAP-4141<https://issues.cask.co/browse/CDAP-4141>`__ -
+  Fixed node.js version checking for the cdap.sh script in the cdap sdk.
+
+- `CDAP-4393<https://issues.cask.co/browse/CDAP-4393>`__ -
+  Fixed the bug in Tags search which didnt' allow user to search for all the tags.
+
+- `CDAP-3695<https://issues.cask.co/browse/CDAP-3695>`__ -
+  No failures occurred in the recent builds. Marking as resolved fix.
+
+- `CDAP-4301<https://issues.cask.co/browse/CDAP-4301>`__ -
+  Added support for CDH5.5
+
+- `CDAP-3900<https://issues.cask.co/browse/CDAP-3900>`__ -
+  Return empty list in case of logs are requested for the deleted program runs.
+
+- `CDAP-3542<https://issues.cask.co/browse/CDAP-3542>`__ -
+  Fixed issue when workflow contains fork and program executing on one of the branches fails because of some error, other concurrently running programs were remaining in the RUNNING state, even if the Workflow is completed.
+
+- `CDAP-4119<https://issues.cask.co/browse/CDAP-4119>`__ -
+  When CDAP master switches from active to standby, the programs that are running are marked failed.
+
+- `CDAP-3345<https://issues.cask.co/browse/CDAP-3345>`__ -
+  Fixed a bug where TTL for datasets were incorrect, reduced by (/1000) after upgrade. After running upgradeTool, please make sure the TTL values of tables are as expected.
+
+- `CDAP-3861<https://issues.cask.co/browse/CDAP-3861>`__ -
+  Fixed an issue about stopping all dependent services when a service is stopped.
+
+- `CDAP-2878<https://issues.cask.co/browse/CDAP-2878>`__ -
+  [Note: Please remove this JIRA from known-issues in Release notes.]
+
+- `CDAP-4045<https://issues.cask.co/browse/CDAP-4045>`__ -
+  A new property master.collect.containers.log has been added to cdap-site.xml, which determines if container logs are streamed back to the cdap-master process log. (This has always been the default behavior). For MapR installations, this must be turned off (set to false).
+
+- `CDAP-4384<https://issues.cask.co/browse/CDAP-4384>`__ -
+  Fixes an issue that prevents streams from being read in HA HDFS mode.
+
+- `CDAP-4373<https://issues.cask.co/browse/CDAP-4373>`__ -
+  "Fixed a problem that prevented MapReduce jobs from being run when the Resource Manager switches from active to standby in a Kerberos-enabled HA cluster."
+
+- `CDAP-4112<https://issues.cask.co/browse/CDAP-4112>`__ -
+  Fixed a bug that prevented applications from using HBase directly.
+
+- `CDAP-4539<https://issues.cask.co/browse/CDAP-4539>`__ -
+  Fixed a bug in the init scripts that forced cdap-kafka-server, cdap-router, and cdap-auth-server to have hive client installed.
+
+- `CDAP-4526<https://issues.cask.co/browse/CDAP-4526>`__ -
+  Fixed init scripts to print service status when stopped
```

Comment:
inconsistent punctuation (period at the end of statement or not).
if its a small number of inconsistencies, maybe make the edits in the JIRA itself?